schema/encoding/jsonschema/all_go16_test.go

```diff
@@ -0,0 +1,11 @@
+// +build !go1.7
+
+package jsonschema_test
```

Comment:
interesting, why does this have a separate package? I'm not familiar with this pattern. Doesn't this mean you can't inspect package variables? Is that the intention? what happens when you run this from the top most level via ./... ?